src/DotNetLightning.Core/Serialization/Msgs/Msgs.fs

```diff
@@ -272,6 +276,11 @@ module ILightningSerializable =
 
             (d :> ILightningSerializable<ErrorMsg>)
                 .Serialize(ls)
+        | :? WarningMsg as d ->
+            ls.Write(TypeFlag.Warning, false)
+
+            (d :> ILightningSerializable<WarningMsg>)
+                .Serialize(ls)
```

Comment:
@webwarrior-ws nit: no need for parentheses in line above

Reply:
again, in line with other code